src/python/python-proto/python_proto/api/graph_query/v1beta1/client.py

```diff
@@ -13,21 +15,32 @@
     QueryGraphWithUidRequest,
     QueryGraphWithUidResponse,
 )
+from python_proto.client import Connectable, GrpcClientConfig
 from python_proto.grapl.common.v1beta1.messages import Uid
 
 
 @dataclass(frozen=True, slots=True)
-class GraphQueryClient:
-    tenant_id: proto_common_msgs.Uuid
+class GraphQueryClient(Connectable):
     proto_client: GraphQueryServiceStub
+    client_config: GrpcClientConfig
+
+    # implements Connectable
+    @classmethod
+    def connect(cls, client_config: GrpcClientConfig) -> GraphQueryClient:
+        address = os.environ["GRAPH_QUERY_CLIENT_ADDRESS"]
+        channel = grpc.insecure_channel(address)
+        stub = GraphQueryServiceStub(channel)
+
+        return cls(proto_client=stub, client_config=client_config)
 
     def query_with_uid(
         self,
+        tenant_id: proto_common_msgs.Uuid,
         node_uid: Uid,
         graph_query: GraphQuery,
     ) -> QueryGraphWithUidResponse:
         request = QueryGraphWithUidRequest(
-            tenant_id=self.tenant_id,
```

Comment:
this was a bad pattern; these `python-proto` clients should be as bone-simple-maps-to-proto as possible, zero smarts